contents/docs/plugins/build/index.md

```diff
@@ -81,6 +51,30 @@ export function onEvent(event, meta) {
 
 This plugin is admittedly useless since PostHog can already show you this information, but it serves to explain how things work. Note how you can choose what kind of events you want to operate on by using the existing event properties.
 
+## Documentation guidelines
+
+To ensure that plugins are useful to others, we require that plugins submitted to [our integration library](/integrations) follow some guidelines. This helps us ensure we describe plugins correctly and give other users the information they need to use a plugin correctly. 
+
+We encourage you to add more info, screenshots, etc., if you feel it is useful, but as a minimum we require that plugins submitted to [hey@posthog.com](mailto:hey@posthog.com?subject=Submit%20Plugin%20to%20Repository&body=Plugin%20GitHub%20link%3A) answer the follow questions. 
```

Comment:
How does someone submit this info...just as text in the email?

Reply:
My bad. Intended to be added as a README.MD. My fault for editing things out of order.